tests/test_convolution.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+from pyfar import Signal
+from pyfar.dsp import convolve_overlap_add
+
+
```

Comment:
How about adding two modes 

**cyclic** would return the result of cyclic convolution. This could be done by computing the full convolution and than adding the tail (the part that exceeds the output of `mode='same'` to the start of the result in a first step and than cuts the tail in a second.

**cut** would be similar to 'same' but the length of the output would be the length of the longest signal, not the first signal.